cmd/kyma/alpha/create/module/module.go

```diff
@@ -32,64 +38,102 @@ func NewCmd(o *Options) *cobra.Command {
+
+You configure the simple mode by providing the "--module-config-file" flag with a config file path.
+The module config file is a YAML file used to configure the following attributes for the module:
+
+- name:         a string, required, the name of the Module
+- version:      a string, required, the version of the Module
+- channel:      a string, required, channel that should be used in the ModuleTemplate
+- manifest:     a string, required, reference to the manifests, must be a relative file name.
+- defaultCR:    a string, optional, reference to a YAML file containing the default CR for the module, must be a relative file name.
+- resourceName: a string, optional, default={NAME}-{CHANNEL}, the name for the ModuleTemplate that will be created
+- security:     a string, optional, name of the security scanners config file
+- internal:     a bool, optional, default=false, determines whether the ModuleTemplate should have the internal flag or not
+- beta:         a bool, optional, default=false, determines whether the ModuleTemplate should have the beta flag or not
+- labels:       a map with string keys and values, optional, additional labels for the generated ModuleTemplate
+- annotations:  a map with string keys and values, optional, additional annotations for the generated ModuleTemplate
+
+The "manifest" and "defaultCR" paths are resolved against the module's directory, as configured with "--path" flag.
+The "manifest" file contains all the module's resources in a single, multi-document YAML file. These resources will be created in the Kyma cluster when the module is activated.
+The "defaultCR" file contain a default Custom Resource for the module that will be installed along with the module.
+It is additionally schema-validated against the Custom Resource Definition for it's Group/Kind that must exists is the set of resources in the "manifest" file.
+
+### Kubebuilder mode configuration
+The Kubebuilder mode is DEPRECATED.
+The Kubebuilder mode is configured automatically if the "--module-config-file" flag is not provided.
+
+In this mode you have to explicitly provide the module name and version using "--name" and "--version" flags, respectively.
+Some defaults, like the module manifest file location and the default CR file location are then resolved automatically, but you can override these with available flags.
+
+### Modules as OCI artifacts
+Modules are built and distributed as OCI artifacts. 
+This command creates a component descriptor in the configured descriptor path (./mod as a default) and packages all the contents on the provided path as an OCI artifact.
+The internal structure of the artifact conforms to the [Open Component Model](https://ocm.software/) scheme version 3.
+
+If you configured the "--registry" flag, the created module is validated and pushed to the configured registry.
+During the validation the default CR resource, if defined, is validated against a corresponding CustomResourceDefinition.
```

Comment:
```suggestion
During the validation the **defaultCR** resource, if defined, is validated against a corresponding CustomResourceDefinition.
```
Do I get it right that it is the same  **defaultCR** from line 66?

Reply:
@mmitoraj yes, it's the resource from that file.